pecan/commands/serve.py

```diff
@@ -64,9 +68,24 @@ def should_reload(self, event):
                         return True
                 return False
 
+            def ignore_events_one_sec(self):
+                if not self.wait:
+                    self.wait = True
+                    t = threading.Thread(target=self.wait_one_sec)
+                    t.start()
+
+            def wait_one_sec(self):
```

Comment:
This seems kind of clunky and susceptible to races. Are you sure you don't just want to wrap the on_modified stuff with a mutex or something? What's the issue that you're experiencing?